docs.feldera.com/docs/pipelines/transactions.md

```diff
@@ -179,6 +179,32 @@ When the status is `TransactionInProgress` or `CommitInProgress`, the `transacti
     </TabItem>
 </Tabs>
 
+## Automatic Transaction Orchestration
+
+In addition to initiating transactions programmatically via the API, certain input connectors can also be configured to initiate transactions automatically.
+This feature enables users to leverage transactions **declaratively** within the pipeline’s SQL definition.
+
+Use cases:
+
+- **Backfill** — When a pipeline starts, input connectors often have historical data available. It is desirable to ingest and process this data as part of a
+  single transaction as explained in the previous sections.
+
+- **Atomic processing of related updates** — Some data sources, such as Delta Lake, group multiple updates into atomic transactions.
+ Processing each such transaction as a single Feldera transaction preserves this atomicity end-to-end.
+
+Multiple connectors may initiate transactions concurrently. In this case, Feldera automatically combines their individual transaction requests into a single coordinated transaction:
+
+- The transaction **begins** when the first connector initiates it.
+- The transaction **commits** when the last participating connector commits the transaction.
+
+For example, if several connectors performing backfill are configured to use transactions, their operations will be merged into one composite transaction, effectively
+equivalent to initiating a single transaction through the REST API that ingests all historical inputs together.
```

Comment:
so a transaction is never broken into pieces, but multiple transactions may be executed as one